src/python/gudhi/datasets/generators/points.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Hind Montassif
+#
+# Copyright (C) 2021 Inria
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+import numpy as np
+import itertools
+
+def _generate_random_points(n_samples, dim):
```

Comment:
There should be a `torus` somewhere in the function name

Reply:
Done.